templates/default/sql/dml/3.2.0.sql.erb

```diff
@@ -1,3 +1,13 @@
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`, `visibility`) VALUES ("max_allowed_long_running_http_requests", "<%= node['hopsworks']['max_allowed_long_running_http_requests'] %>", 1);
 
 REPLACE INTO `hopsworks`.`variables`(`id`, `value`) VALUES ("hdfs_file_op_job_util", "hdfs:///user/<%= node['hadoop_spark']['user'] %>/hdfs_file_operations-<%= node['hadoop_spark']['hopsworks_jobs_py']['version'] %>.py");
+
+INSERT INTO `hopsworks`.`users`
+(`username`, `password`, `email`, `fname`, `lname`, `activated`, `title`, `false_login`, `isonline`, `secret`, `validation_key`, `mode`, `password_changed`, `notes`, `status`, `max_num_projects`, `two_factor`, `tours_state` , `salt`) VALUES
+('flyingduck', '<%= @flyingduck_password %>', '<%= node['flyingduck']['hopsworks']['email'] %>', 'Flyingduck', 'Server', '2021-03-16 16:17:00', 'Mr', -1, 0, 'V3WBPS4G2WMQ53VA', NULL, 0, '2021-03-16 16:17:00', NULL, 2, 30, 0, 3, '<%= @flyingduck_salt %>')
+ON DUPLICATE KEY UPDATE `username`="flyingduck";
+
+INSERT IGNORE INTO `hopsworks`.`user_group` (`uid`, `gid`) SELECT `uid`, 1007 FROM `hopsworks`.`users` WHERE `username`='flyingduck';
```

Comment:
Why does flyingduck need a user in Hopsworks? I don't see any Hopsworks PR using this user.